### DISCUSSION TITLE
pointT (PCL) like structure in open3D

### DISCUSSION THREAD

**Discussion by brleme** [isl-org/Open3D#5946] [General]
Hello all,

I am new to open3D and I was wondering if it has something similar to pointT structure (PCL). I am using multispectral imaging and I need two or three more channels besides RGB. Is it possible to add information for a point in open3D?

Thank you!

**Comment by yuecideng** [comment]
Hi, you can use `open3d.t.geometry.PointCloud`. Please see the [tutorial](http://www.open3d.org/docs/latest/tutorial/t_geometry/pointcloud.html#Point-cloud-attributes-setter-and-getter) for reference.

**Reply by brleme** [reply]
Thank you!!! solved my problem.
